management/schemas/job.yaml

```diff
@@ -1,23 +1,135 @@
 schemas:
+  Override:
+    type: object
+    properties:
+      source:
+        type: string
+      target:
+        type: array
+        items:
+          type: string
+      op:
+        type: string
+        enum: ['constant', 'literal', 'stringify', 'stringify-array']
 
-  JobInput:
+  CreateAdsFromProductsTask:
     type: object
-    required:
-      - TaskId
     properties:
       TaskId:
         type: string
-        maxLength: 100
+        enum: ['create-ads-from-products']
       TaskArgs:
         type: object
-      PartitionId:
-        type: string
+        properties:
+          ProductUris:
+            type: array
+            items:
+              type: string
+              format: uri
+          AdTemplateUri:
+            type: string
+            format: uri
+          FlightId:
+            type: integer
+            format: int32
+          Overrides:
+            type: object
+            description: 'A map of ProductUri keys with Override values'
+            additionalProperties:
+              $ref: '#/schemas/Override'
 
-  JobOutput:
```

Comment:
The removal of this schema definition means the reference in `management/schemas/flight.yaml` doesn't resolve anymore.

```
Jobs:
      type: array
      items:
        $ref: './job.yaml#/schemas/JobOutput'
```

Reply:
Thanks for flagging this one. That is now `Job`.